_includes/site/layout/docs/tabs.html

```diff
@@ -1,23 +1,46 @@
 <div class="row docs-nav">
-  <div class="col-md-9">
+  <div class="col-md-8">
     <ul class="nav nav-tabs visible-md-block visible-lg-block">
-      {% for tab in site.data.doc_categories %}
-        <li {% if page.category == tab.title %}class="active"{% endif %}>
-          <a href="{{ site.baseurl }}/{{ tab.href }}">
+      {% assign release_version = page.path | split:"/" | slice:1 | first %}
+      {% for tab in site.data.doc_categories[release_version] %}
+          {% assign tab_title_with_version = tab.title | append:" (" | append:release_version | append:")" %}
+          <li {% if page.category == tab_title_with_version %}class="active"{% endif %}>
+          <a href="{{ site.baseurl }}/{{ tab.href | replace:'$$VERSION$$',release_version }}">
             {{ tab.title }}
           </a>
         </li>
       {% endfor %}
     </ul>
     <ol class="breadcrumb visible-xs-block visible-sm-block">
-      <li><a href="{{ site.baseurl }}/docs/">Docs</a></li>
+      <li><a href="{{ site.baseurl }}/docs/{{ release_version }}/">Docs</a></li>
       <li class="active">{{ page.category }}</li>
     </ol>
   </div>
-  <div class="col-md-3 edit-on-github visible-md-block visible-lg-block text-right">
-    {% assign filename = page.path | split:"/" | last | downcase %}
-    <a class="icon github" href="{{ site.github.repository_url }}/edit/forsetisecurity.org/{{ page.path }}">
-      Edit this page
-    </a>
+  <div class="col-md-4 text-right">
+    <div class="btn-group" id="version-dropdown" style="margin-top: 1px; margin-bottom: 0; margin-right: 30px">
+      <a data-target="#version-dropdown" class="btn btn-raised dropdown-toggle" data-toggle="dropdown" aria-expanded="false">
+        {{ release_version }}
+        <span class="caret"></span>
+      </a>
+      <ul class="dropdown-menu">
+        {% assign doc_name = page.path | split:"/" | slice:2,999 | join:"/" %}
+        {% assign url_name = page.url | split:"/" | slice:3,999 | join:"/" | remove:"index.html" %}
+        
+        {% assign versions = site.data.doc_version_index[doc_name] | unshift:'latest' %}
+        {% unless versions %}
+          {% assign versions = 'latest' | split:' ' %}
+        {% endunless %}
+
+        {% for version in versions %}
+          <li {% if release_version == version %}class="active"{% endif %}><a href="{{ site.baseurl }}/docs/{{ version }}/{{ url_name }}">{{ version }}</a></li>
+        {% endfor %}
+      </ul>
+    </div>
+    <div class="edit-on-github visible-lg-inline-block visible-md-inline-block">
+      {% assign filename = page.path | split:"/" | last | downcase %}
+      <a class="icon github" href="{{ site.github.repository_url }}/edit/forsetisecurity.org/{{ page.path }}">
+        Edit this page
+      </a>
+    </div>
   </div>
-</div>
+</div>
```

Comment:
Final newline please.

Reply:
Newlines addressed in forthcoming update.